### PR TITLE
Add soil efflux and remote sensing data

### PR DESCRIPTION
**Pull request by atkinsjeff**
i have also corrected some previous variable information

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -12,13 +12,91 @@
...
+#'   \item{dateTime}{date and time of sample in YYYY-MM-DD HH:MM:SS}
+#'   \item{subplotID}{subplot identifier}
+#'   \item{run}{differnt sampling occurence}
+#'   \item{nestedPlot}{sampling point of collar inside of subplot}
+#'   \item{soilCO2Efflux}{soil carbon dioxide efflux in umol CO2 m^2 s^-1}
+#'   \item{soilTemp}{soil temperature at 7 cm depth in celsius}
+#'   \item{VWC}{volumetric water content from 0 to 20 cm depth}
+#'   \item{notes}{additional information}
+#' }
+#'  @source \url{http://atkinsjeff.github.io}
```
Comment (bpbond) [inline R/data.R]: Extra space/tab here is messing up help page rendering.

Comment (bpbond) [reply R/data.R]: Plus, is it right that soil respiration should have this `source`?

File: R/data.R [hunk 2]
```diff
@@ -12,13 +12,91 @@
...
+#'   \item{sd.return.ht}{standard deviation of lidar return distances}
+#'   \item{sky.fraction}{ratio of sky hits to lidar returns}
+#'   \item{cover.fraction}{1/sky fraction}
+#'   \item{max.ht}{same as max can ht, removed in later forestr updates}
+#'   \item{scan.density}{no. of lidar returns divided by transect length}
+#'   \item{rumple}{outer surface variability divided by transect lenght}
+#'   \item{clumping.index}{clumpiness}
+#'   \item{enl}{effective number of layers}
+#' }
+#'  @source \url{http://atkinsjeff.github.io}
```
Comment (bpbond) [inline R/data.R]: Also doesn't render correctly.

File: R/data.R [hunk 3]
```diff
@@ -12,13 +12,91 @@
...
+#'
+#' @format A data frame with four variables: \code{dateTime} \code{subplotID}, \code{run},\code{nestedPlot},
+#'   \code{soilCO2Efflux}, \code{soilTemp}, \code{VWC} and \code{notes}
+#'
+#' \describe{
+#'   \item{dateTime}{date and time of sample in YYYY-MM-DD HH:MM:SS}
+#'   \item{subplotID}{subplot identifier}
+#'   \item{run}{differnt sampling occurence}
+#'   \item{nestedPlot}{sampling point of collar inside of subplot}
+#'   \item{soilCO2Efflux}{soil carbon dioxide efflux in umol CO2 m^2 s^-1}
```
Comment (bpbond) [inline R/data.R]: Use mu (µ) not u

**Comment by atkinsjeff** [comment]
here is the latest one.

**Comment by atkinsjeff** [comment]
adding more and more data! This time includes the remote sensing data and removing the old data stuff.

**Comment by atkinsjeff** [comment]
I am still missing some tests to add to this, but this adds a lot more stuff and hopefully will pass everything...it passes all local checks now.

**Comment by atkinsjeff** [comment]
well....all checks fail. nice.

**Comment by atkinsjeff** [comment]
I think this one works... :\

**Comment by bpbond** [comment]
Look at all your test code! 👏

**Comment by bpbond** [comment]
I have cleaned up the merge conflicts.

`fd_inventory()` has been changed (in b3905fc00f0) so that it now returns, along with the inventory, biomass information for each tree. This creates at least two problems:

- the metadata test is now failing (see #20 )
- Mission creep. I'd suggest that `fd_inventory()` should just return what its name says: the inventory

If we want to provide biomass data (do we?!? Unless these allometry coefficients are FoRTE-measured, probably not?) let's have a separate `fd_biomass()` or something.

Thoughts @atkinsjeff ? Thanks

**Comment by atkinsjeff** [comment]
They are regionally specific but not essential, no.  I can make an fd_biomass.R easily though.

**Comment by bpbond** [comment]
I'd much prefer to _keep things simple_. In particular, I would vote that for now we're not going to do 'derived' data, just primary data actually measured by the FoRTE team. So could we save biomass for fortedata v2?

**Comment by atkinsjeff** [comment]
Deal. Let's cut it.

**Comment by bpbond** [comment]
@atkinsjeff FYI you had added a file `inst/extdata/fd_ceptometer.csv` but your code was

```R
  par <- read_csv_file("fd_ceptometer.CSV")
```
Note file extension difference. This works on a Mac, but not on Linux, which is why Travis tests were failing.

**Pull request merged**